privacy_evaluator/attacks/sample_attack.py

```diff
@@ -1,8 +1,7 @@
-from typing import Tuple
 import numpy as np
 
-from privacy_evaluator.attacks.attack import Attack
-from privacy_evaluator.classifiers.classifier import Classifier
+from .attack import Attack
+from ..classifiers.classifier import Classifier
 
 
 class SampleAttack(Attack):
```

Comment:
Could you provide more documentation here? What is the purpose of this class? What will happen here?

Reply:
The initial purpose was to provide an example for developers on how to implement an attack. I would say that it is now not really needed anymore and is rather confusing. Should remove the class?

Reply:
i think it can be removed

Reply:
Just removed it.